src/DisjointAwarePipeline.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
Don't think these comments should be here?

Reply:
Done